.gitignore

```diff
@@ -50,6 +53,11 @@ queries*.smt2
 transcript
 dump
 
+# F* build infrastructure.
+extracted/
+.vs/
+.ionide/
+
```

Comment:
Can you please remove these editor setting, as discussed on #2508, they can be added locally if needed. Also `extracted`, I don't see F* generating this directory.

Reply:
Okay. I will remove that.

Reply:
Extracted is generated after you run `make -C src ulib-in-fsharp` and this is excluded `ulib/fs/generated` folder which is required to build `ulib.fsproj` . I would say `extracted` is generic pattern in this repository to produce extracted files so that's why I place it in the ignore folder.

Reply:
Can you please move it to `ulib/` (as is done with the `ml/extracted`)? Note that the directory name is not mandated by F*, it is a command line option and so each "user" of extraction, e.g. `ulib`, sets it. There are instances in `examples` where we use `_output`, and that is ignored locally in `examples/.gitignore`.